src/models/FormEditMetadata.ts

```diff
@@ -8,17 +8,20 @@ export const validationSchema = Yup.object().shape({
     .min(4, (param) => `Title must be at least ${param.min} characters`)
     .required('Required'),
   description: Yup.string().required('Required').min(10),
+  price: Yup.number().required('Required'),
   links: Yup.array<EditableMetadataLinks[]>().nullable(),
   timeout: Yup.string().required('Required')
 })
 
 export function getInitialValues(
   metadata: MetadataMarket,
-  timeout: number
+  timeout: number,
+  price: number
 ): Partial<MetadataPublishForm> {
```

Comment:
almost sure this typing here is wrong and should be `MetadataEditForm`. This should also make all those errors go away